contracts/src/oracle/Oracle.cairo

```diff
@@ -55,8 +55,8 @@ func get_spot_entries_for_sources{syscall_ptr: felt*, pedersen_ptr: HashBuiltin*
 func get_spot_entries{syscall_ptr: felt*, pedersen_ptr: HashBuiltin*, range_check_ptr}(
     pair_id: felt
 ) -> (entries_len: felt, entries: SpotEntry*) {
-    let (sources) = alloc();
-    return get_spot_entries_for_sources(pair_id, 0, sources);
+    let (all_sources_len, all_sources) = Oracle.get_all_sources(pair_id);
```

Comment:
How did this pass tests before?

Reply:
I guess we didn't have any tests for the `get_spot_entries` path and the test you added fixes that